src/elements/content-sidebar/versions/VersionsSidebarContainer.js

```diff
@@ -54,7 +54,9 @@ class VersionsSidebarContainer extends React.Component<Props, State> {
 
         // Forward the current version id that is passed in via the wrapping route
         if (prevVersionId !== versionId) {
-            onVersionChange(versionId);
+            const { versions } = this.state;
+            const isCurrentVersion = versionId === versions[0].id;
+            onVersionChange(versionId, isCurrentVersion);
```

Comment:
I think we should consider passing another callback here as a param. The callback param could be used by the host app to completely exit the version history preview. 

I'd also suggest moving `isCurrentVersion` to a property in an object to clean up the method signature.

Reply:
I think the behavior on close is to actual go back to previewing the current version.
Either way a callback could be helpful here. My other approach was for the consumer to call preview.show() on the file again which would reset to the current version.

Reply:
I don't think the consumer has direct access or control over when the `preview.show()` method gets called. Even if it did, that could get pretty tricky given the internal state tracking in Elements.